handlers/getSignedUrlV1.js

```diff
@@ -1,17 +1,17 @@
 const { Storage } = require('@google-cloud/storage');
-const { getJsonFrom } = require('../common/api_adapter');
-const { bondBaseUrl, promiseHandler, Response, samBaseUrl, determineBondProvider } = require('../common/helpers');
+const apiAdapter = require('../common/api_adapter');
+const { bondBaseUrl, promiseHandler, Response, samBaseUrl, determineBondProvider, BondProviders } = require('../common/helpers');
 
 const getSignedUrlV1 = promiseHandler(async (req) => {
     const { bucket, object, dataObjectUri } = req.body || {};
     const auth = req.headers.authorization;
     const provider = dataObjectUri && determineBondProvider(dataObjectUri);
     try {
-        const credentials = provider ?
-            (await getJsonFrom(`${bondBaseUrl()}/api/link/v1/${provider}/serviceaccount/key`, auth)).data :
-            await getJsonFrom(`${samBaseUrl()}/api/google/v1/user/petServiceAccount/key`, auth);
+        const credentials = (provider && provider !== BondProviders.HCA) ?
+            (await apiAdapter.getJsonFrom(`${bondBaseUrl()}/api/link/v1/${provider}/serviceaccount/key`, auth)).data :
+            await apiAdapter.getJsonFrom(`${samBaseUrl()}/api/google/v1/user/petServiceAccount/key`, auth);
         const storage = new Storage({ credentials });
-        const [url] = await storage.bucket(bucket).file(object).getSignedUrl({
+        const url = await storage.bucket(bucket).file(object).getSignedUrl({
```

Comment:
This doesn't look right. The library call returns a single element array that needs to be destructured.

Reply:
Thanks! True, indeed:
```
export declare type GetSignedUrlResponse = [string];
export interface GetSignedUrlCallback {
    (err: Error | null, url?: string): void;
}
```

Albeit unexpected, at least to me.
I restored the original line in 'getSignedUrlV1` and updated the unit test mock value to be an array of string instead of string.